lxc/network.go

```diff
@@ -72,6 +72,10 @@ func (c *cmdNetwork) Command() *cobra.Command {
 	networkListCmd := cmdNetworkList{global: c.global, network: c}
 	cmd.AddCommand(networkListCmd.Command())
 
+	// List allocations
+	networkListAllocationsCmd := cmdNetworkListAllocations{global: c.global, network: c}
```

Comment:
@gabrielmougard we'll need some docs added for this new command.

@ru-fu any preference where we add it?

Reply:
> @gabrielmougard we'll need some docs added for this new command.
> 
> @ru-fu any preference where we add it?

From a quick glance, that looks like we need a new "How to display IPAM information" (or similar) under [Networking](https://documentation.ubuntu.com/lxd/en/latest/networks/).
Is this for all types of managed networks? Or also for network devices, or only for bridge/OVN?

Reply:
@ru-fu this displays networking information for all types of managed network but also instances attached to them, network forwards and network load-balancers

Reply:
Yes, then this should be the right location.